src/molecules/ContentBlock/ContentBlock.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import cn from 'classnames';
+import { Banner, Heading, Button } from '../../index';
+
+export interface Block {
+  type?: 'ContentBlock';
+  __type?: 'ContentBlock';
+  title: string;
+  text: string;
+  image: {
+    title: string;
+    url: string;
+  };
+  orientation: boolean;
+  buttonLink?: string;
+  buttonText?: string;
+}
+
+export interface ContentBlockProps {
+  block: Block;
+  className?: string;
+  search?: string;
+  whiteBackground?: boolean;
+}
+
+export const ContentBlock: React.FC<ContentBlockProps> = ({
+  className,
+  block,
+  search,
+  whiteBackground = false,
+}: ContentBlockProps) => (
```

Comment:
```suggestion
}) => (
```

type of the props is implicit when you`re using `React.FC<ContentBlockProps>`